vendor/bat-native-ads/src/bat/ads/internal/catalog_os_info.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BAT_ADS_INTERNAL_CATALOG_OS_INFO_H_
+#define BAT_ADS_INTERNAL_CATALOG_OS_INFO_H_
+
+#include <string>
+
+namespace ads {
+
+struct OsInfo {
+  OsInfo();
+  explicit OsInfo(const OsInfo& info);
```

Comment:
Please remove explicit, we will be removed from other structures as we refactor/change code

Reply:
@tmancey I removed it from day_part_info.h as well